src/modules/transaction/components/StakeItem/StakeItem.js

```diff
@@ -17,11 +17,11 @@ import styles from './StakeItem.css';
  * @param {String} title text to use instead of the address e.g. validator username
  * @param {Boolean} truncate text to use instead of the address e.g. validator username
  */
-const StakeItem = ({ stake, address, title, truncate, token }) => {
+const StakeItem = ({ stake, address, title, truncate, token, reward }) => {
   const accountPath = routes.explorer.path;
 
   return (
-    <span className={styles.container}>
+    <span className={`${styles.container} stake-item`}>
```

Comment:
`stake-item` seems to only be used for testing purposes? Maybe we can use test id instead. Confusing to add classes just for testing purposes.